src/plugins/faultManagement/FaultManagementListItem.vue

```diff
@@ -0,0 +1,206 @@
+    props: {
+        fault: {
+            type: Object,
+            required: true
+        },
+        isSelected: {
+            type: Boolean,
+            default: () => {
+                return false;
+            }
+        }
+    },
+    data() {
+        return {
+        };
+    },
+    computed: {
+        acknowledged() {
+            return this.fault?.acknowledged;
+        },
+        liveValue() {
+            return this.fault?.parameterDetail?.currentValue?.engValue?.doubleValue;
+        },
+        liveValueClassname() {
+            const currentValue = this.fault?.parameterDetail?.currentValue;
+            if (!currentValue || currentValue.monitoringResult === 'IN_LIMITS') {
+                return '';
+            }
+
+            let classname = this.getRangeConditionBasedClassname(currentValue.rangeCondition);
+            classname += ' ';
+            classname += this.getRangeMonitoringResultClassname(currentValue.monitoringResult);
+
+            return classname.trim();
+        },
+        name() {
+            return `${this.fault?.id?.name}/${this.fault?.id?.namespace}`;
+        },
+        severity() {
+            return this.fault?.severity;
+        },
+        triggerTime() {
+            return this.fault?.triggerTime;
+        },
+        triggerValue() {
+            return this.fault?.parameterDetail?.triggerValue?.engValue?.doubleValue;
+        },
+        tripValueClassname() {
+            const triggerValue = this.fault?.parameterDetail?.triggerValue;
+            if (!triggerValue || triggerValue.monitoringResult === 'IN_LIMITS') {
+                return '';
+            }
+
+            let classname = this.getRangeConditionBasedClassname(triggerValue.rangeCondition);
+            classname += ' ';
+            classname += this.getRangeMonitoringResultClassname(triggerValue.monitoringResult);
+
+            return classname.trim();
+        }
+    },
+    watch: {
+    },
+    mounted() {
+    },
+    beforeDestroy() {
+    },
+    methods: {
+        getRangeConditionBasedClassname(rangeCondition) {
+            if (rangeCondition === 'LOW') {
+                return 'is-limit--lwr';
+            }
+
+            if (rangeCondition === 'HIGH') {
+                return 'is-limit--upr';
+            }
+
+            return '';
+        },
+        getRangeMonitoringResultClassname(monitoringResult) {
+            if (monitoringResult === 'CRITICAL') {
+                return 'is-limit--red';
+            }
+
+            if (monitoringResult === 'WARNING') {
+                return 'is-limit--yellow';
+            }
+
+            if (monitoringResult === 'WATCH') {
+                return 'is-limit--cyan';
+            }
+
+            return '';
+        },
+        toggleSelected(event) {
+            const faultData = {
+                fault: this.fault,
+                selected: event.target.checked
+            };
+
+            event.target.checked = !event.target.checked;
```

Comment:
We're toggling the checked boolean value after assigning it to faultData. Is this intentional?

Reply:
ahh good catch, we dont need to change `event.target.checked` as it is driven by `isSelected`